app/javascript/components/tree-view/style.scss

```diff
@@ -0,0 +1,5 @@
+.react-tree-view {
+    .opacity {
```

Comment:
Do we use this anywhere?

Reply:
If not, we don't even have to load the file :thinking:

Reply:
Yes, we do, but I forgot where.

Reply:
In this case, could we move this to a location which is not included through React? Maybe `app/assets/stylesheets/miq_tree.scss`? That file is automatically loaded into our global stylesheet.